app/_data/docs_nav_gateway_2.8.x.yml

```diff
@@ -5,8 +5,10 @@
       url: /gateway/
       absolute_url: true
     - text: Version Support Policy
-      url: /konnect-platform/support-policy
+      url: /gateway/latest/support-policy
```

Comment:
Not sure why, but this one still results in a 404

Reply:
Maybe it should be `/gateway/{{page.kong_version}}/support-policy` because it looks like the file is in `/app/gateway/2.8.x/support-policy`